orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -94,27 +96,23 @@ class OperationsController {
 
     def augmentedContext = [trigger: pipeline.trigger]
     def processedPipeline = contextParameterProcessor.process(pipeline, augmentedContext, false)
+    processedPipeline.trigger = objectMapper.convertValue(processedPipeline.trigger, Trigger)
 
     if (pipelineError == null) {
       startPipeline(processedPipeline)
     } else {
+      throw pipelineError
```

Comment:
Why are we now throwing the error before calling `markPipelineFailed`?  This will effectively revert the change made in #1937 as we'll no longer be logging errors parsing pipelines and will just throw a server-side error.  (Also, everything after this line is unreachable.)

Reply:
Whoops, that was left around from debugging a test, I thought that was backed out already. Will remove